### PR TITLE
Update release notes for 2.0.0 release

### LINKED ISSUE #163: [Task] Update release notes
**Is your task related to a problem? Please describe.**
The release notes section of the documentation needs an update to reflect all the major changes that have been done since BAMM 1.0.0.

### PR DESCRIPTION
**Pull request by atextor**
Fixes #163

### CODE REVIEW

**Review by georgschmidtdumont — APPROVED** [review]

File: src/docs/modules/appendix/pages/release-notes.adoc [hunk 1]
```diff
@@ -17,13 +17,29 @@ SPDX-License-Identifier: MPL-2.0
...
 === Release 2.0.0
 
-- Introduces Entity inheritance. The new *Abstract Entity* Meta Model Element and *extends* attribute may be used to
-define a hierarchy of Entities in an Aspect Model. The concept of extending an Entity replaces the existing concept of
-refining an Entity.
+* Introduces Entity inheritance. The new *Abstract Entity* Meta Model Element and *extends*
+attribute may be used to define a hierarchy of Entities in an Aspect Model. The concept of extending
+an Entity replaces the existing concept of refining an Entity. The `bamm:refines` keyword was
+therefore removed as its semantics are superseded by Entity inheritance.
+* The `bamm:name` attribute has been removed, as it encoded redundant information. The "short" name
```
Comment (chris-volk) [inline src/docs/modules/appendix/pages/release-notes.adoc:24]: Alternative: Remove the `bamm:name` attribute to reduce redundancy. (...)

File: src/docs/modules/appendix/pages/release-notes.adoc [hunk 2]
```diff
@@ -17,13 +17,29 @@ SPDX-License-Identifier: MPL-2.0
 [[bamm-2.0.0]]
 === Release 2.0.0
 
-- Introduces Entity inheritance. The new *Abstract Entity* Meta Model Element and *extends* attribute may be used to
-define a hierarchy of Entities in an Aspect Model. The concept of extending an Entity replaces the existing concept of
-refining an Entity.
+* Introduces Entity inheritance. The new *Abstract Entity* Meta Model Element and *extends*
```
Comment (chris-volk) [inline src/docs/modules/appendix/pages/release-notes.adoc:20]: Introduce Entity inheritance, the according Meta Model Element *Abstract Entity* and *Abstract Property* as well as the *extends* attribute to define a hierarchy of Entities. This replaces the concept of refining an entity and the `bamm:refines` attribute in order to reduce complexity and modeling efforts.

File: src/docs/modules/appendix/pages/release-notes.adoc [hunk 3]
```diff
@@ -17,13 +17,29 @@ SPDX-License-Identifier: MPL-2.0
...
+attribute may be used to define a hierarchy of Entities in an Aspect Model. The concept of extending
+an Entity replaces the existing concept of refining an Entity. The `bamm:refines` keyword was
+therefore removed as its semantics are superseded by Entity inheritance.
+* The `bamm:name` attribute has been removed, as it encoded redundant information. The "short" name
+  of an element is the local name part of its URI, i.e., the part following the `#` symbol.
 * The meta model elements `Unit`, `QuantityKinds` and their corresponding attributes were moved from
```
Comment (chris-volk) [inline src/docs/modules/appendix/pages/release-notes.adoc:26]: The meta model elements `Unit`, `QuantityKinds` and their corresponding attributes have been moved from `unit` to the `bamm` name space. The `unit` namespace is now solely reserved for the units of the xref:bamm-specification:appendix:unitcatalog.adoc#unit-catalog-units[unit catalog] itself.

File: src/docs/modules/appendix/pages/release-notes.adoc [hunk 4]
```diff
@@ -17,13 +17,29 @@ SPDX-License-Identifier: MPL-2.0
...
   the `unit` namespace to the `bamm` namespace. This fixes an inconsistency because the `unit`
   namespace is only intended for the units of the
   xref:bamm-specification:appendix:unitcatalog.adoc#unit-catalog-units[unit catalog].
+* The
+   https://github.com/OpenManufacturingPlatform/sds-bamm-aspect-meta-model/blob/main/src/main/resources/bamm/unit/2.0.0/units.ttl[formal
+   description (units.ttl)] of the units catalog is now automatically generated from its original
+   source (see xref:bamm-specification:ROOT:units.adoc[Units]) and the
+   https://github.com/OpenManufacturingPlatform/sds-bamm-aspect-meta-model/blob/main/buildSrc/src/main/resources/custom-units.ttl[local adjustments].
+* The `localeCode` and `languageCode` attributes were moved from the `bamm` namespace to the
```
Comment (chris-volk) [inline src/docs/modules/appendix/pages/release-notes.adoc:35]: have been

File: src/docs/modules/appendix/pages/release-notes.adoc [hunk 5]
```diff
@@ -17,13 +17,29 @@ SPDX-License-Identifier: MPL-2.0
 [[bamm-2.0.0]]
 === Release 2.0.0
```
Comment (chris-volk) [inline src/docs/modules/appendix/pages/release-notes.adoc:18]: Would it make sense to also include the documentation updates in the release notes or is this just a bonus that we do not mention?

Comment (chris-volk) [reply src/docs/modules/appendix/pages/release-notes.adoc:18]: Would it also make sense to mention that with the BAMM 2.0.0 the SHACL shapes and validation have been further improved?

Comment (chris-volk) [reply src/docs/modules/appendix/pages/release-notes.adoc:18]: Ah, specification is mentioned. Would it make sense to mention it in the beginning (it is not the most significant one, but it is one that is easy to understand and skip to the next point)

**Pull request merged**